imaging/tools/custom/customMouseWheelScrollTool.ts

```diff
@@ -230,6 +232,15 @@ export default class CustomMouseWheelScrollTool extends BaseTool {
         this.nextIndex = validIndex;
         // Scroll to the calculated index
         scrollToIndex(element, validIndex);
+        if (getActiveLayer(element.id)) {
+          let activeLayer = getActiveLayer(element.id);
+          let layers = cornerstone.getLayers(element);
+          let newActiveLayer =
+            //@ts-ignore
+            activeLayer.layerId == layers[0].layerId ? layers[1] : layers[0];
+          //@ts-ignore
+          setActiveLayer(element.id, newActiveLayer.layerId);
+        }
```

Comment:
Cosa ritorna la chiamata nel caso in cui non ci siano layers?

Reply:
undefined